CHANGELOG.md

```diff
@@ -36,6 +36,9 @@ Converting from v3.x to v4.0
   other components they depend on (instead of string properties):
   - ContactGeometry (ContactSphere, ContactHalfSpace, ContactMesh)
 - Many of the methods in ScaleTool have now been marked const.
+- We created a new unified command line interface that will replace the
+  single-tool command line executables (`scale`, `ik`, `id`, `rra`, `cmc`,
+  etc.).
```

Comment:
Perhaps indicate the change users will need to make if they're currently calling the command-line executables from their code.